### ISSUE TITLE
[BUG] Unused variables in tag_group module

### ISSUE THREAD

**Issue by robin-checkmk** [Checkmk/ansible-collection-tribe29.checkmk#332]
**Describe the bug**

See the screenshot below for unused variables in the `tag_group` module.

**Component Name**

`tag_group`

**Ansible Version**

```console
$ ansible --version

```

**Checkmk Version and Edition**

```console
e.g. 2.X.YpZ (CRE)
```

**Collection Version**

```console
$ ansible-galaxy collection list
# /home/robin/.ansible/collections/ansible_collections
Collection        Version
----------------- -------
tribe29.checkmk   0.21.0 
```

**To Reproduce**
N/A

**Expected behavior**

No used variables can be found.

**Actual behavior**

Used variables are reported.

**Minimum reproduction example**

```yaml

```

**Additional context**

![image](https://user-images.githubusercontent.com/93658105/237062871-5b8be203-51ae-4a16-bed7-cbefa34586a9.png)
